src/main/java/subway/line/LineResponse.java

```diff
@@ -0,0 +1,38 @@
+package subway.line;
+
+import subway.station.StationResponse;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class LineResponse {
+    private final Long id;
+    private final String name;
+    private final String color;
+    private final int distance;
+    private final List<StationResponse> stations;
+
+    public LineResponse(Long id, String name, String color, int distance, List<StationResponse> stations) {
+        this.id = id;
+        this.name = name;
+        this.color = color;
+        this.distance = distance;
+        this.stations = stations;
+    }
+
+    public static LineResponse createLineResponse(Line line) {
```

Comment:
리턴 타입이 있으니 메서드명은 간결하게 작성해도 좋을 것 같네요

```suggestion
    public static LineResponse create(Line line) {
```